lib/src/start.js

```diff
@@ -106,7 +106,11 @@ function getRequestType ({ pathname, fileType }, { proxyPrefix, pushstate }) {
  * This function takes in the path from the request and the model and returns the needed values to finish running the request.
  **/
 function parseUrl (pathname, model) {
-  const fileType = pipe(path.extname, mime.getType)(pathname)
+  let fileType = pipe(path.extname, mime.getType)(pathname)
+  if (fileType === null) {
+    // prevent static files with query strings having a null fileType and thus returning the wrong request type from getRequestType
+    fileType = pipe(path.extname, mime.getType)(removeQueryString(pathname))
```

Comment:
I believe we can assume that removing the query params directly is fine, since `?` should be encoded if the character is used in the name of the static file, meaning that the following should be good (if statement not needed):
```
function parseUrl (pathname, model) {
   const fileType = pipe(path.extname, mime.getType)(removeQueryString(pathname))
   const requestType = getRequestType({ pathname, fileType }, model)
   ....
```